rasa/engine/training/graph_trainer.py

```diff
@@ -48,6 +49,7 @@ def train(
         output_filename: Path,
         force_retraining: bool = False,
         is_finetuning: bool = False,
+        training_type: TrainingType = TrainingType.BOTH,
```

Comment:
wondering if we have to many params here - e.g. `is_finetuning` is also related to the `ModelMetadata`. The alternative would be that the `Recipe` returns schemas _and_ a `ModelMetadata` object, what do you think?